app/src/test/java/org/simple/clinic/util/RxErrorsRule.kt

```diff
@@ -0,0 +1,53 @@
+package org.simple.clinic.util
```

Comment:
I also found a couple of test classes where this should have been added, but wasn't.

1. `BruteForceProtectionTest`
2. `RecyclerViewUserScrollDetectorTest`

Did you intentionally not add this rule to this classes?

Reply:
No, I think I missed these two classes. Fixing this.